components/ContestFilter.tsx

```diff
@@ -21,6 +24,15 @@ export default function ContestFilter({
 
   const filterList = ['Open', 'All', 'Voting', 'Closed']
 
+  const [showContestCreationModal, setshowContestCreationModal] =
+    useState<boolean>(false)
+
+  const handleOpenCreateContestForm = () => {
```

Comment:
call this toggle instead of handle, so it makes it more obvious what this is doing